extra/release_tool.py

```diff
@@ -3085,72 +3085,6 @@ def do_hosted_release(version=None):
     print("Tags for release %s successfully created" % version)
 
 
-def do_verify_integration_references(args, optional_too):
-    int_dir = integration_dir()
-    problem = False
-
-    repos = Component.get_components_of_type("git", only_release=(not optional_too))
-
-    for repo in repos:
-        # integration is not checked, since the current checkout records the
-        # version of that one.
-        if repo.git() == "integration":
-            continue
-
-        # Try some common locations.
-        paths = ["..", "../go/src/github.com/mendersoftware"]
-        path = find_repo_path(repo.git(), paths)
-
-        if path is None:
-            print("%s not found. Tried: %s" % (repo.git(), ", ".join(paths)))
-            sys.exit(2)
-
-        revs = figure_out_checked_out_revision(None, path)
```

Comment:
You can also remove this function. Not used anywhere else, AFAICS.

Reply:
Good catch, thank you.